src/usdb_syncer/gui/song_table/song_table.py

```diff
@@ -299,6 +302,13 @@ def stage_local_songs(self, directory: str) -> None:
         self._stage_rows(rows)
         _logger.info(f"Added {len(rows)} songs to batch.")
 
+    def stage_song_ids(self, song_ids: list[SongId]) -> None:
+        self._stage_rows(row for id in song_ids if (row := self._model.row_for_id(id)))
```

Comment:
This does not work for row 0.

Reply:
ab oops did not think of that. Thanks a lot